### ISSUE TITLE
Support docker run commands on pod deploy

### ISSUE THREAD

**Issue by mydatascience** [lf-edge/eden#588]
We don't support all commands for run - e.g. webthings require -v and we don't have this as-is

**Comment by rvs** [comment]
True, but some of the docker run options would be pretty difficult to express with EVE -- would be nice to see a list of what maps well and what doesn't
